src/js/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -58,14 +51,7 @@ class Post911GIBStatusApp extends React.Component {
                   print a copy of this screen for benefit and eligibility verification.
                 </p>
               </div>
-              <UserInfoSection enrollmentData={enrollmentData}/>
-              <EnrollmentHistory enrollmentData={enrollmentData}/>
-              <div className="feature help-desk">
-                <h2>Need help?</h2>
-                <div>Call the Vets.gov Help Desk</div>
-                <div>1-855-574-7286</div>
-                <div>Monday - Friday, 8:00am - 8:00pm (ET)</div>
-              </div>
+              <Main/>
```

Comment:
I tried to make this a child route, so that I could call `{this.props.children}` here instead of explicitly rendering this particular component, but for some reason ran into problems when I tried to do it that way. This works for now.